backend/model/school.js

```diff
@@ -0,0 +1,58 @@
+const Student = require("./student");
+const Class = require("./Class")
+
+
+
+
+class School {
+  constructor() {
+    this.classes ={
+      
+    }
+   
+    
+  }
+
+
+   addClass(name, teacher) {
+    let newClass = new Class(name, teacher);
+    this.classes[name] = newClass;
+    return newClass
+  }
+
+
+  enrollStudent(className, student) {
+    let newStudent = new Student(student.name, student.age, student.city, student.grade)
+    if(this.classes[className]){
```

Comment:
It's good to check if the class exists, but you should do it in your router/server as a middleware function. So that if the class does exist - you can send an appropriate response message to display to the user "Class Astronomy already exists in our system"